### DISCUSSION TITLE
How to add additional data to data object in import action lifecycle method beforeCreate?

### DISCUSSION THREAD

**Discussion by NaungYeHtet** [filamentphp/filament#12241] [Help]
### Package

Actions

### Package Version

v3.0.0

### How can we help you?

AgentImporter class
```
protected function beforeCreate(): void
{
        $this->data['code'] = Str::random(10);
}
```

<img width="752" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/ea94810b-33b8-4a5e-a1df-f83eb588016b">

**Comment by leandrocfe** [comment]
Did you add 'code' in the `$fillable` Model?

**Reply by NaungYeHtet** [reply]
I have Model::unguard(); in service provider but do I need to add fillable?

**Reply by NaungYeHtet** [reply]
Added fillable but still got error
